package.json

```diff
@@ -49,7 +49,7 @@
     "mjolnir.js": "^2.2.0",
     "prop-types": "^15.7.2",
     "react-virtualized-auto-sizer": "^1.0.2",
-    "viewport-mercator-project": ">=6.2.1"
+    "viewport-mercator-project": "^6.2.3 || ^7.0.1"
```

Comment:
Can't use `@math.gl/web-mercator@^3.1.3`?

Reply:
- We have documentation that tell users to import `WebMercatorViewport` from `viewport-mercator-project`. Removing the dependency will be a breaking change.
- When used with deck.gl v7.x, `viewport-mercator-project@6.x` is the shared dependency.